orchestrator/cli/domain_gen_helpers/helpers.py

```diff
@@ -27,29 +27,29 @@ def generic_mapper(prop_name: str, model_diffs: Dict[str, Dict[str, Set[str]]])
     return {k: {val[1] for val in v} for k, v in grouped}
 
 
-def map_create_fixed_inputs(model_diffs: Dict[str, Dict[str, Set[str]]]) -> Dict[str, Set[str]]:
+def map_create_fixed_inputs(model_diffs: dict[str, dict[str, set[str]]]) -> dict[str, set[str]]:
```

Comment:
Couple of type aliases might someday help to make this typing a bit more readable.

Reply:
Agree, I don't really know what is passed along here